src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,37 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
-    for i in range(0, len(arr) - 1):
+    for i in range(0, len(arr)):
         cur_index = i
         smallest_index = cur_index
+        # print(arr)
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
-        # TO-DO: swap
-
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index+1, len(arr)):
+            if (arr[j] < arr[cur_index]):
+                smallest_index = j
+                # TO-DO: swap
+                temp = arr[smallest_index]
+                arr[smallest_index] = arr[cur_index]
+                arr[cur_index] = temp
 
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
-def bubble_sort( arr ):
-
+def bubble_sort(arr):
+    is_swapped = True
+    while (is_swapped):
+        is_swapped = False
+        for i in range(0, len(arr)-1):
+            if (arr[i] > arr[i+1]):
+                temp = arr[i]
+                arr[i] = arr[i+1]
+                arr[i+1] = temp
+                is_swapped = True
     return arr
```

Comment:
nicely done, there is also an optimized bubble, which sorts on the way back as well, something you could look into implementing if you have free time.